packages/website/src/pages/layout/EllipsisText.tsx

```diff
@@ -11,8 +16,18 @@ const Page = () => (
         description="Allows to display text that will automatically truncate and shows a tooltip on hover if it overflows the max width."
         id="EllipsisText"
         propsMetadata={EllipsisTextMetadata}
-        demo={<EllipsisTextDemo />}
-    />
+        demo={<EllipsisTextDefaultLongDemo />}
+    >
+        <div className="plasma-page-layout__section">
+            <Stack>
+                <EllipsisTextDefaultLongDemo title="Default ellipsis" />
+                <EllipsisTextDefaultShortDemo title="Default no ellipsis" />
+                <EllipsisTextNoWrapContainerDemo title="Default in no wrap container" />
+                <EllipsisTextLineClampLongDemo title="Line clamp ellipsis" />
+                <EllipsisTextLineClampShortDemo title="Line clamp no ellipsis" />
+            </Stack>
+        </div>
+    </PageLayout>
```

Comment:
We usually use the `examples` props instead of a children

```suggestion
    examples={{
            long: <EllipsisTextDefaultLongDemo title="Server-side table integrated with @tanstack/react-query"/>,
            short: <EllipsisTextDefaultShortDemo title="Default no ellipsis" />,
            noWrapContainer: <EllipsisTextNoWrapContainerDemo title="Default in no wrap container" />,
            lineClampLong: <EllipsisTextLineClampLongDemo title="Line clamp ellipsis" />,
            lineClampShort: <EllipsisTextLineClampShortDemo title="Line clamp no ellipsis" />
    />
```